.env.dev

```diff
@@ -0,0 +1,9 @@
+DATA_FOLDER=data-dev
+
+FUSEKI_PORT=3030
+ENDPOINT=http://fuseki:3030
+
+RDF_DIFFER_API_PORT=3040
```

Comment:
This variable seems to be the only one relevant for the prod environment. The other ones are bound to Fuseki service.